src/main.js

```diff
@@ -33,12 +33,17 @@ if (!isFirefoxPanel()) {
 
 window.appStore = store;
 
+window.selectors = selectors;
+
 // Expose the bound actions so external things can do things like
 // selecting a source.
-window.actions = {
-  selectSource: actions.selectSource,
-  selectSourceURL: actions.selectSourceURL
-};
+// window.actions = {
+//   selectSource: actions.selectSource,
+//   selectSourceURL: actions.selectSourceURL
+// };
+
+// JDE: temporarily expose all actions for integration tests.
+window.actions = actions;
```

Comment:
hmm, making all of these globals is somewhat sad :/

perhaps we add a function `getGlobalsForTesting` that returns `{ store, selectors, action }`